spec/factories/registration.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :registration do
+    user { association :user }
```

Comment:
Writing only user and event should work, it will find factories with the same name
```
FactoryBot.define do
  factory :registration do
    user
    event
  end
end
```